### ISSUE TITLE
Cannot save in google colab

### ISSUE THREAD

**Issue by testerpce** [araffin/rl-baselines-zoo#19]
This is my google colab file

https://colab.research.google.com/drive/1dnROnz1kDQsHI4ReTjjF79EnHc3GlvKd

For some reason I cannot seem to save any trained model in google colab.
It is giving some kind of segmentation error. The initial part of the file is the same as the template google colab file and the version of stable baselines is the latest as I am updating the stable baselines version. 

However on trying to save it is giving this error.
!python train.py --algo ppo2 --env BipedalWalkerHardcore-v2 -f logs -n 10000
========== BipedalWalkerHardcore-v2 ==========
OrderedDict([('cliprange', 'lin_0.2'),
             ('ent_coef', 0.001),
             ('gamma', 0.99),
             ('lam', 0.95),
             ('learning_rate', 'lin_2.5e-4'),
             ('n_envs', 16),
             ('n_steps', 2048),
             ('n_timesteps', 100000000.0),
             ('nminibatches', 32),
             ('noptepochs', 10),
             ('normalize', True),
             ('policy', 'MlpPolicy')])
Using 16 environments
Normalizing input and return
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/stable_baselines/common/policies.py:436: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Saving to logs/ppo2/BipedalWalkerHardcore-v2_2
[e88f4df2d52c:03067] *** Process received signal ***
[e88f4df2d52c:03067] Signal: Segmentation fault (11)
[e88f4df2d52c:03067] Signal code: Address not mapped (1)
[e88f4df2d52c:03067] Failing at address: 0x7fee1aba120d
[e88f4df2d52c:03067] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7fee1de55890]
[e88f4df2d52c:03067] [ 1] /lib/x86_64-linux-gnu/libc.so.6(getenv+0xa5)[0x7fee1da94785]
[e88f4df2d52c:03067] [ 2] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(_ZN13TCMallocGuardD1Ev+0x34)[0x7fee1e2ffe44]
[e88f4df2d52c:03067] [ 3] /lib/x86_64-linux-gnu/libc.so.6(__cxa_finalize+0xf5)[0x7fee1da95615]
[e88f4df2d52c:03067] [ 4] /usr/lib/x86_64-linux-gnu/libtcmalloc.so.4(+0x13cb3)[0x7fee1e2fdcb3]
[e88f4df2d52c:03067] *** End of error message ***

I want to save the logs folder on to my google drive or at least find a way to download them into my computer. But in this case I cannot even seem to save the file onto log it seems.

**Comment by araffin** [comment]
Hello,

>Saving to logs/ppo2/BipedalWalkerHardcore-v2_2

looking at your stack trace, the model was saved in that folder.
I should add a warning, even though you get a seg fault, saving is working.
